src/server_manager/model/digitalocean_account.ts

```diff
@@ -0,0 +1,155 @@
+// Copyright 2018 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {DigitalOceanSession, DropletInfo} from '../cloud/digitalocean_api';
+import * as crypto from '../infrastructure/crypto';
+import {LocalStorageRepository} from '../infrastructure/repository';
+import * as do_install_script from '../install_scripts/do_install_script';
+import {DigitalOceanServer, GetCityId} from '../web_app/digitalocean_server';
+
+import * as account from './account';
+import * as cloud_provider from './cloud_provider';
+import * as server from './server';
+
+const SHADOWBOX_TAG = 'shadowbox';
+const MACHINE_SIZE = 's-1vcpu-1gb';
+
+export class DigitalOceanAccount implements account.Account {
```

Comment:
Let's define a DigitalOceanAccount interface in the model, and have the implementation in the app code.

The /model directory must not have any dependencies on application code.